src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1273,7 +1273,9 @@ internal enum ErrorCode
         ERR_BadPdbData = 8036,
         // available 8037-8039
         INF_UnableToLoadSomeTypesInAnalyzer = 8040,
-        // available 8041-8049
+        WRN_LoadedAnalyzerDiffers = 8041,
+        WRN_MissingAnalyzerDependency = 8042,
```

Comment:
Do we need to do something to make the new warnings not configurable (so that they don't show up in sln explorer / ruleset editor)? (I am not sure whether other warnings / errors related to loading / running of analyzer are configurable)